engine/routes.py

```diff
@@ -14,9 +14,16 @@
 from util.common import (StreamEngineException, TimedOutException, MissingDataException,
                          MissingTimeException, ntp_to_datestring, StreamKey, InvalidPathException)
 from util.san import onload_netCDF, SAN_netcdf
+from util.releasenotes import ReleaseNotes
 
 log = logging.getLogger(__name__)
 
+release = ReleaseNotes()
+log.info('Starting {} v{} {} ({})'.format(
```

Comment:
I'm not sure why we are reading and parsing the release notes file more than once. It seems like the formatting, except for the "Starting", could be done in default.py and assigned to a variable for use at this point.

Reply:
this is something that also bugged me when I wrote it.  I have gone back and added in an instance method for obtaining a singleton of the class.